packages/kit/src/runtime/app/env.js

```diff
@@ -14,5 +14,9 @@ export const mode = import.meta.env.MODE;
  * @type {import('$app/env').amp}
  */
 export const amp = !!import.meta.env.VITE_SVELTEKIT_AMP;
+/**
+ * @type {import('$app/env').version}
+ */
+export const version = import.meta.env.VITE_APP_VERSION?.toString();
```

Comment:
it's guaranteed to exist and be a string

```suggestion
export const version = import.meta.env.VITE_APP_VERSION;
```

Reply:
ah, I see — it's not being set in dev. I think it's probably easier to set it to something dev-specific in dev, that way we have fewer points of divergence